docs/basics/command-line.md

```diff
@@ -0,0 +1,171 @@
+---
+title: "Command-line"
+description: "A simple walkthrough of how to perform basic IPFS operations using the command-line."
+---
+
+# Basic CLI Operations
+
+This short guide aims to walk you through the basics of using IPFS with the CLI (Command Line Interface). You will learn how to add, retrieve, read, and remove files from within the CLI. We will try to keep it simple here, if you are unsure about the meaning of some terms you can check out the [glossary](../concepts/glossary/). Also don't forget to check out the rest of the [docs](../) to learn more about IPFS!
+
+All instructions and examples shown here were performed and tested on an M1 Mac, the IPFS commands will be same on macOS, Windows, and Linux. You will need to know how to navigate through your computers directories from within the CLI, if you are unsure how to do that, it is recommended to learn how before continuing with this guide.
+
+## Install IPFS using the CLI
+
+Installing IPFS using the CLI is simple, and there is already a great guide on how to do so [here](../install/command-line/).
```

Comment:
Don't like to pages using `here`. Use the link to tell me what's on that page:

```
We've got a great guide on [using the CLI to interact with IPFS](../install/command-line).
```

Something like that.